client/components/jetpack/scan-threats-new/index.tsx

```diff
@@ -0,0 +1,347 @@
+		.sort( ( a, b ) => b.severity - a.severity );
+	const lowSeverityThreats = threats.filter( ( threat ) => threat.severity < 3 );
+
+	return (
+		<>
+			<Card>
+				<SecurityIcon icon={ securityIcon } />
+				<h1 className="scan-threats-new scan__header">{ headerMessage }</h1>
+				<p className="scan-threats-new__header-message">
+					{ headerSummary }{ ' ' }
+					{ maxSeverity < 3 && (
+						<SupportInfo
+							position="top"
+							text={ translate(
+								"Low risk items don't have a negative impact on your site and can be safely ignored."
+							) }
+						/>
+					) }
+				</p>
+				{ hasFixableThreats && (
+					<>
+						<div className="scan-threats-new__buttons">
+							<p>{ fixSummary }</p>
+							<Button
+								primary
+								className="scan-threats-new__fix-all-threats-button"
+								onClick={ openFixAllThreatsDialog }
+								disabled={ ! hasFixableThreats || updatingThreats.length > 0 }
+							>
+								{ translate( 'Auto fix all' ) }
+							</Button>
+						</div>
+					</>
+				) }
+			</Card>
+			<ThreatListHeader />
+			<div className="scan-threats-new__threats">
+				{ highSeverityThreats &&
+					highSeverityThreats.map( ( threat ) => (
+						<ThreatItem
+							key={ threat.id }
+							threat={ threat }
+							onFixThreat={ () => openDialog( 'fix', threat ) }
+							onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
+							isFixing={ isFixing( threat ) }
+							contactSupportUrl={ contactSupportUrl( site.URL ) }
+							isPlaceholder={ false }
+						/>
+					) ) }
+				{ ! highSeverityThreats &&
+					lowSeverityThreats &&
+					lowSeverityThreats.map( ( threat ) => (
+						<ThreatItem
+							key={ threat.id }
+							threat={ threat }
+							onFixThreat={ () => openDialog( 'fix', threat ) }
+							onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
+							isFixing={ isFixing( threat ) }
+							contactSupportUrl={ contactSupportUrl( site.URL ) }
+							isPlaceholder={ false }
+						/>
+					) ) }
+				<div className="scan-threats-new__low-risk">
+					<FoldableCard header={ <ThreatLowRiskItemHeader threatCount={ countMap.low } /> }>
+						{ highSeverityThreats &&
+							lowSeverityThreats &&
+							lowSeverityThreats.map( ( threat ) => (
+								<ThreatItem
+									key={ threat.id }
+									threat={ threat }
+									onFixThreat={ () => openDialog( 'fix', threat ) }
+									onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
+									isFixing={ isFixing( threat ) }
+									contactSupportUrl={ contactSupportUrl( site.URL ) }
+									isPlaceholder={ false }
+								/>
+							) ) }
+					</FoldableCard>
+				</div>
+			</div>
+
+			{ ! error && (
+				<div className="scan-threats-new__rerun">
+					<p className="scan-threats-new__rerun-help">
+						{ translate(
+							'If you have manually fixed any of the threats listed above, you can {{button}}run a manual scan now{{/button}} or wait for Jetpack to scan your site later today.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'If you have manually fixed any of the threats above, you can {{button}}run a scan now{{/button}} or wait for Jetpack to scan your site later today.' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/more-informative-scan/threat-list)